msteams-platform/resources/moodleInstructions.md

```diff
@@ -32,49 +32,49 @@ In order to install and configure this application you'll need:
 
 ## Step 1: Install the Microsoft 365 Moodle Plugins
```

Comment:
Other topics (for example: Get Started) numbers the steps in the following format: "1. Install the Microsoft 365 Moodle Plugins" instead of Step 1 and Step 2....We need to maintain consistency across the document. Please check how it is written in other places.